src/main/java/de/mediathekview/mlib/compression/CompressionManager.java

```diff
@@ -41,7 +40,7 @@ public OutputStream compress(final CompressionType aCompressionType,
       final OutputStream aOutputStream) throws IOException {
     switch (aCompressionType) {
       case XZ:
-        return new XZOutputStream(aOutputStream, new LZMA2Options(LZMA2Options.PRESET_MAX));
+        return new XZCompressorOutputStream(aOutputStream, 9);
       case GZIP:
         return new GZIPOutputStream(aOutputStream) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use another way to initialize this instance. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3599)